crypto/cipher.go

```diff
@@ -0,0 +1,46 @@
+package crypto
+
+import (
+	"bytes"
+	"crypto/aes"
+	"errors"
+	"github.com/btcsuite/btcd/btcec"
+)
+
+var errInvalidPadding = errors.New("invalid PKCS#7 padding")
+
+// Implement PKCS#7 padding with block size of 16 (AES block size).
+
+// AddPKCSPadding adds padding to a block of data
+func AddPKCSPadding(src []byte) []byte {
+	padding := aes.BlockSize - len(src)%aes.BlockSize
+	padtext := bytes.Repeat([]byte{byte(padding)}, padding)
+	return append(src, padtext...)
+}
+
+// RemovePKCSPadding removes padding from data that was added with addPKCSPadding
+func RemovePKCSPadding(src []byte) ([]byte, error) {
+	length := len(src)
+	padLength := int(src[length-1])
+	if padLength > aes.BlockSize || length < aes.BlockSize {
+		return nil, errInvalidPadding
+	}
+
+	return src[:length-padLength], nil
+}
+
+// EncryptAndSign(inputPublicKey, inData) MAIN PROCEDURE:
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function EncryptAndSign should be of the form "EncryptAndSign ..."